docs/docs/migration-guide.mdx

```diff
@@ -29,6 +29,17 @@ how you can migrate from one version to another.
   component in your `policies` configuration (`config.yml`) you can replace it
   with `TEDPolicy`. It accepts the same configuration parameters.
 
+* [Conversation sessions](domain.mdx#session-configuration) are now enabled by default
```

Comment:
The title of the targeted section is currently `Session Configuration`. I'd vote for renaming this to `Conversation Sessions`.

Reply:
@erohmensing ?

Reply:
Mm, I think it makes the most sense to have the title of the section to reflect the high level keys in the domain

Reply:
Okay, then let's leave it as it is.